evalml/pipelines/utils.py

```diff
@@ -83,14 +83,13 @@ def _get_drop_all_null(X, y, problem_type, estimator_class, sampler_name=None):
 def _get_replace_null(X, y, problem_type, estimator_class, sampler_name=None):
     component = []
     all_nullable_cols = X.ww.select(
-        ["IntegerNullable", "AgeNullable", "BooleanNullable"],
+        ["IntegerNullable", "AgeNullable"],
         return_schema=True,
     ).columns
     nullable_target = isinstance(
         y.ww.logical_type,
         (
             logical_types.AgeNullable,
-            logical_types.BooleanNullable,
```

Comment:
so should we actually just remove the logic replacing `BooleanNullable` in the `ReplaceNullableTypes` transformer? I'm pretty sure that if we encountered `AgeNullable` or `IntegerNullable`, we would add the `ReplaceNullableTypes` transformer and end up replacing `BooleanNullable` anyway.